api/scripts/create-or-update-sco-organizations.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-console */
+// Usage: BASE_URL=... PIXMASTER_EMAIL=... PIXMASTER_PASSWORD=... node create-or-update-sco-organizations.js path/file.csv
+
+'use strict';
+require('dotenv').config();
+const request = require('request-promise-native');
+
+const { assertFileValidity, parseCsv } = require('./helpers/csvHelpers');
+
+const baseUrl = process.env.BASE_URL || 'http://localhost:3000';
+
+function organizeOrganizationsByExternalId(data) {
+  const organizationsByExternalId = {};
+
+  data.forEach((organization) => {
+    if (organization.attributes['external-id']) {
+      organizationsByExternalId[organization.attributes['external-id']] = { id: organization.id, ...organization.attributes };
+    }
+  });
+
+  return organizationsByExternalId;
+}
+
+async function createOrUpdateOrganizations({ accessToken, organizationsByExternalId, csvData }) {
+  for (let i = 0; i < csvData.length; i++) {
+    if (require.main === module) {
+      process.stdout.write(`${i + 1}/${csvData.length}`);
+    }
+
+    const [externalId, name] = csvData[i];
+
+    if (!(externalId && name)) return;
+
+    const existingOrganization = organizationsByExternalId[externalId];
```

Comment:
attention ici il faut s'assurer que `externalId` est traité de la même manière que ceux déjà insérés.
Les orgas ayant déjà un `externalId`, l'ont en majuscules (cf script `update-sco-organizations-with-province-code-and-external-id.js`). Donc ici, il faut `.toUpperCase()`.

Reply:
Bien vu !